### ISSUE TITLE
GitHub links broken

### ISSUE THREAD

**Issue by jarofgreen** [modxcms/docs#231]
Go To 

https://docs.modx.com/revolution/2.x/developing-in-modx/advanced-development/custom-manager-pages/custom-manager-pages-tutorial

Scroll to the bottom, the Comment Policy block.

Click the "Suggest an Edit" link.

You will be taken to 

https://github.com/modxcms/docs/issues/new?title=%5BEdit%20Suggestion%5D%20Customizing+the+Manager&body=%23%23+Page+Title%0ACustomizing+the+Manager%0A%0A%23%23+URL%0Ahttps%3A%2F%2Fdocs.modx.com%2Frevolution%2F2.x%2Fadministering-your-site%2Fcustomizing-the-manager%0A%0A%23%23+Issue+or+Suggestion

Note "Customizing+the+Manager" and the URL in the resulting body - https://docs.modx.com/revolution/2.x/administering-your-site/customizing-the-manager

Basically, the "Suggest an Edit" link is for the wrong page!

**Comment by Mark-H** [comment]
@jaygilmore Could you take a look at this? I think that's out of reach for most people with access to the docs.
